crates/krun-server/src/server.rs

```diff
@@ -52,7 +50,7 @@ fn handle_connection(mut stream: TcpStream) -> Result<()> {
     let mut envs: HashMap<String, String> = env::vars().collect();
 
     let launch = read_request(&stream)?;
-    envs.extend(launch.envs);
+    envs.extend(launch.env);
```

Comment:
Is there a reason for doing this? Doesn't this mean the user cannot have precise control over the env vars for a specific invocation of `krun`?

Reply:
Running graphical apps requires a number of `envs` exported by `sommelier`. The idea is that users are able to export _additional_ environment variables, as otherwise the UX would be pretty bad. If there's a demand for that, we could add another option in the future to just export those variables.

Reply:
Yeah, I kinda realized this after a while... Haha.